apps/omg_watcher/lib/omg_watcher/db/txoutput.ex

```diff
@@ -141,7 +141,9 @@ defmodule OMG.Watcher.DB.TxOutput do
       _ =
         DB.TxOutput
         |> where(blknum: ^blknum, txindex: ^txindex, oindex: ^oindex)
-        |> Repo.update_all(set: [spending_tx_oindex: spending_oindex, spending_txhash: spending_txhash])
+        |> Repo.update_all(
+          set: [spending_tx_oindex: spending_oindex, spending_txhash: spending_txhash, updated_at: DateTime.utc_now()]
```

Comment:
It looks like ecto supports `timestamps` in combination with `changesets` - am I right?

Reply:
Yes it does.